### PR TITLE
[Nucleus] Fix tests add progress fields

### PR DESCRIPTION
**Pull request by bihanjiang**
Adding job_progress, completed_steps, and total_steps for python api tests.

[[ch171858]](https://app.clubhouse.io/scaleai/story/171858)

Depends on https://github.com/scaleapi/scaleapi/pull/28943

### CODE REVIEW

**Comment by ardila** [comment]
Can you please clarify if you have things passing locally and need to merge this and the dependent PR, or if you are still working on this?

**Comment by bihanjiang** [comment]
> Can you please clarify if you have things passing locally and need to merge this and the dependent PR, or if you are still working on this?

Hey! I'm still working on this, just got back on wifi :')

**Review by ardila — APPROVED** [review]

**Comment by ardila** [comment]
You will have to wait for the other pr to merge before merging this.

@pytest.mark.skip(reason="Times out consistently") for the test you want to skip

**Pull request merged**
